### DISCUSSION TITLE
AI與戰爭上的機器們_幽靈蝙蝠的出現

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#526] [General]
在烏俄大戰的現在，自己對於戰爭上不論是型態或是技術的更新都會稍微留意一下。這次的這篇幽靈蝙蝠就是戰機和AI的結合，並即將投如量產和實際應用。這次的戰機從概念到命名儀式僅僅花費了3年多，這種無人戰機採用AI來自動規劃飛行路線和戰鬥。對於澳洲來說，這是他們自二戰以來首次開發並組裝軍用飛機，而命名也是為了凸顯這類型戰機團結作戰的風格。也許在不久的將來就會看到這類型的戰機投入到實際應用當中。
[https://technews.tw/2022/03/23/boeing-loyal-wingman-officially-named-mq28a-ghost-bat/](https://technews.tw/2022/03/23/boeing-loyal-wingman-officially-named-mq28a-ghost-bat/)

**Comment by Jay-722** [comment]
無人機可以自己飛行自己戰鬥，在戰爭中可能會少犧牲一些人吧

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望不要再發生戰爭了。

**Comment by Sazmafuji** [comment]
我只願戰爭不要再發生，此類技術的增長，希望有更 多的討論

**Comment by sean1325418** [comment]
只能說戰爭是科技發展的催化劑，很多科技都是為了戰爭才發展的

**Comment by xkllkx** [comment]
雖然戰爭不是個好東西，但說不定經過這場戰爭過後，無人機技術大幅進步。

**Comment by PoooYen** [comment]
之後就是機器人打仗了，搞不好以後的職業軍人就是那些CS的明星選手哈哈

**Comment by cordydeng** [comment]
文中有說到可以用在探測方面的用途，相信這種發明不只可以用在軍事上面，在其他方面一定可以有所貢獻
